NERODevelopment/src/models/raspberry_model.h

```diff
@@ -75,6 +75,10 @@ class RaspberryModel : public Model {
   std::optional<bool> getIsTalking() override;
   std::optional<int> getNumberOfCriticalFaults() override;
   std::optional<int> getNumberOfNonCriticalFaults() override;
+  std::optional<float> getFastestTime() override;
```

Comment:
swap these for the getTime function, you also need to implement that in the mock model, this shouldnt be compiling rn?